tests/xtimer_msg_receive_timeout/main.c

```diff
@@ -30,27 +26,27 @@
 #include "xtimer.h"
 #include "timex.h"
 
-#define TEST_PERIOD (100000LU)
+#define TEST_PERIOD     (1*US_PER_SEC) /* 1 sec in US */
```

Comment:
Btw, does the test still work if you lower the time to 100ms? Having automatic testing in mind, the shorter, the better, so the shortest to actually show any bugs would be perfect.